src/js/modules/infragistics.ui.editors.js

```diff
@@ -11351,6 +11409,10 @@
 				if (currentInputValue) {
 					this._editorInput.val(currentInputValue);
 				}
+			} finally {
+				if (this.options.suppressKeyboard) {

```

Comment:
I think you can safely call delete here even without the check - just to make sure the flag always gets removed (even if option is changes by some unexpected handling)